src/java.desktop/share/classes/javax/swing/plaf/synth/SynthTabbedPaneUI.java

```diff
@@ -737,6 +759,22 @@ private void paintContentBorder(SynthContext ss, Graphics g,
               h -= (y - insets.top);
         }
         SynthLookAndFeel.updateSubregion(ss, g, new Rectangle(x, y, w, h));
+
+        if (tabPane.getTabCount() > 0 && (contentOpaque || tabPane.isOpaque())) {
+            // Fill region behind content area
+            Color color = UIManager.getColor("TabbedPane.contentAreaColor");
+            if (color != null) {
+                g.setColor(color);
+            } else if (selectColor == null || selectedIndex == -1) {
+                g.setColor(tabPane.getBackground());
+            } else {
+                g.setColor(selectColor);
+            }
+        }
+
+        // fill content area rect for both GTK and Nimbus LAF here
+        g.fillRect(x, y, w, h);
```

Comment:
shouldn't it be within if block as is done in BasicTabbedPaneUI so that we fill only if it's opaque